src/util/assertAccessible.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Assertion function for enforcing the inclusion of a label on the component
+ *
+ * @param componentName Name of the component which needs a label (.displayName)
+ * @param labelProps list of props to check for a label value
+ */
+export function assertHasLabel(
+  componentName: string,
+  ...labelProps: (undefined | string)[]
+): void {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
I still kinda like wrapping the calling of `assertHasLabel` in the `if (process.env.NODE_ENV !== 'production') {...}` block so the code and import gets stripped from production builds. But this is a very small distinction and I understand the tradeoff of not wanting to repeat having to type of `if (process.env.NODE_ENV !== 'production')`